lib/neo4j/http/node_client.rb

```diff
@@ -26,7 +28,9 @@ def upsert_node(node)
         results.first&.fetch("node")
       end
 
-      def delete_node(node)
+      def delete_node(node, unwind: nil)
```

Comment:
Should we instead create plural methods of these?  So that` unwind:` isn't an optional argument, but instead you're explicitly calling a method to do things in bulk.